siteComponents/utils.css

```diff
@@ -1,3 +1,86 @@
 .mgn-left-auto {
     margin-left: auto;
-}
\ No newline at end of file
+}
+
+
+
+/* Prod Horizontal Card */
+
+.dui-card-prod-hzntl {
+    width: fit-content;
+    height: auto;
+
+    background: var(--dui-light-theme-bg);
+    display: grid;
+
+    grid-auto-columns: 4fr 4fr 3fr 3fr 3fr 3fr;
+    grid-template-areas:
+        'cardImg  cardImg  cardInfo     cardInfo     cardInfo     cardInfo'
+        'cardImg  cardImg  cardDesc     cardDesc     cardDesc     cardDesc'
+        'cardImg  cardImg  cardActions  cardActions  cardActions  cardActions';
+}
+
+.dui-card-prod-hzntl>.dui-badge {
+    width: 5rem;
+    border-radius: 0;
+    top: 1rem;
+    left: 1rem;
+}
+
+.dui-card-prod-hzntl__img-container {
+    grid-area: cardImg;
+    width: 100%;
+    height: 100%;
+    padding: 2rem;
+    display: grid;
+}
+
+.dui-card-prod-hzntl__img {
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
```

Comment:
height by default is 100%

Reply:
As far as I know, height by default is fit-content.